persistence/sql/timeouts_connection_sqlpersistence_[,7).partial.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Timeouts Persister
+component: SqlPersistence
+reviewed: 2020-02-27
+versions: '[4,)'
```

Comment:
why does this need a version setting and it doesn't seem to match the partial's version range?

Reply:
because....it doesn't :) I'll uppate.